### PR TITLE
Update Log, OS, UUID, Random BBEs

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by daneshk** [comment]
@praneesha Can you review please

File: examples/logging-with-context/logging_with_context.bal [hunk 1]
```diff
@@ -3,10 +3,8 @@ import ballerina/random;
 import ballerina/time;
 
 public function main() {
-    // The Ballerina log API provides functions to log at four levels, which are
-    // `DEBUG`, `ERROR`, `INFO`, and `WARN`.
-    // You can pass key/value pairs where the values are function pointers.
-    // These functions can return values, which change dynamically.
+    // The Ballerina log API provides functions to log at four levels, which are `DEBUG`, `ERROR`, `INFO`, and `WARN`.
+    // You can pass key/value pairs where the values are function pointers. These functions can return values, which change dynamically.
```
Comment (praneesha) [inline examples/logging-with-context/logging_with_context.bal:7]:
```suggestion
    // You can pass key/value pairs of which the values are function pointers. These functions can return values, which change dynamically.
```

File: examples/logging-with-context/logging_with_context.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-The Ballerina Log API contains the application log handling functions.<br/><br/>
-For more information on the underlying module,
-see the [Log module](https://docs.central.ballerina.io/ballerina/log/latest/).
+The `log` library contains the application log handling functions.
+
+For more information on the underlying module, see the [Log module](https://docs.central.ballerina.io/ballerina/log/latest/).
 
 ::: code logging_with_context.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/logging-with-context/logging_with_context.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```

File: examples/random-numbers/random_numbers.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
-The `random` library provides functions related to random number generation.<br/><br/>
-For more information on the underlying module,
-see the [Random module](https://docs.central.ballerina.io/ballerina/random/latest/).
+The `random` library provides functions related to random number generation.
+
+For more information on the underlying module, see the [Random module](https://docs.central.ballerina.io/ballerina/random/latest/).
 
 ::: code random_numbers.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/random-numbers/random_numbers.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```

File: examples/uuid-generation/uuid_generation.md [hunk 4]
```diff
@@ -1,9 +1,11 @@
 # Generate UUID
 
-The `uuid` library provides functions related to UUID(Universal Unique Identifier).<br/><br/>
-For more information on the underlying module,
-see the [UUID module](https://docs.central.ballerina.io/ballerina/uuid/latest/).
+The `uuid` library provides functions related to UUID(Universal Unique Identifier).
```
Comment (praneesha) [inline examples/uuid-generation/uuid_generation.md:3]:
```suggestion
The `uuid` library provides functions related to UUIDs (Universal Unique Identifiers).
```

File: examples/uuid-generation/uuid_generation.md [hunk 5]
```diff
@@ -1,9 +1,11 @@
...
+
+For more information on the underlying module, see the [UUID module](https://docs.central.ballerina.io/ballerina/uuid/latest/).
 
 ::: code uuid_generation.bal :::
 
+To run this sample use the `bal run` command.
```
Comment (praneesha) [inline examples/uuid-generation/uuid_generation.md:9]:
```suggestion
To run this sample, use the `bal run` command.
```

File: examples/uuid-operations/uuid_operations.md [hunk 6]
```diff
@@ -1,9 +1,11 @@
 # UUID operations
 
-The `uuid` library provides functions related to UUID(Universal Unique Identifier).<br/><br/>
-For more information on the underlying module,
-see the [UUID module](https://docs.central.ballerina.io/ballerina/uuid/latest/).
+The `uuid` library provides functions related to UUID(Universal Unique Identifier).
```
Comment (praneesha) [inline examples/uuid-operations/uuid_operations.md:3]:
```suggestion
The `uuid` library provides functions related to UUIDs (Universal Unique Identifiers).
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**